process/block/shardblock.go

```diff
@@ -484,6 +484,11 @@ func (sp *shardProcessor) restoreMetaBlockIntoPool(miniBlockHashes map[string]ui
 			continue
 		}
 
+		processedMiniBlocks := metaBlock.GetMiniBlockHeadersWithDst(sp.shardCoordinator.SelfId())
```

Comment:
I would use the same variable name as it is used in line 526: crossMiniBlockHashes as actually this is why it represents